src/main/webapp/WEB-INF/tags/ksr/securityRequestRole.tag

```diff
@@ -8,12 +8,18 @@
 <c:set var="genericAttributes" value="${DataDictionary.AttributeReferenceDummy.attributes}" />
 <c:set var="securityRequestRole" value="${KualiForm.document.securityRequestRoles[securityRequestRoleIndex]}" />
 <c:set var="roleTitle" value="${securityRequestRole.roleInfo.id} : ${securityRequestRole.roleInfo.namespaceCode} - ${securityRequestRole.roleInfo.name}"/>
+<c:set var="roleInquiryUrl" value="${ConfigProperties.application.url}/inquiry.do?methodToCall=start&businessObjectClassName=org.kuali.kfs.kim.impl.role.Role&id=${securityRequestRole.roleInfo.id}&mode=standalone"/>
 
 <h3>&nbsp;</h3>
 <table width="100%" border="0" cellpadding="0" cellspacing="0" class="datatable">
   <tr>
     <th colspan="2" width="100%">
-      <b><c:out value="${roleTitle}"/></b>
+      <b>
+        <c:out value="${roleTitle}"/>
+        <a href="<c:out value='${roleInquiryUrl}'/>" target="_blank" title="Open in new tab" class="new-window" onclick="event.stopPropagation();">

```

Comment:
in the interest of being nice to screen readers, maybe the anchor title should include the role name, something like "Open details for role ROLE-NAME in a new tab"
Also, a space between the c:out and the anchor might be good

Reply:
I don't think the other KFS new-tab-link anchors include extra info about what's being linked to, but I agree that it would be helpful to include that. I'll see if I can adjust the title accordingly.

I'll also add a non-breaking space between the printed title and the link.